circle.yml

```diff
@@ -0,0 +1,76 @@
+machine:
+    java:
+        version: oraclejdk8
+    environment:
+        ANDROID_HOME: /usr/local/android-sdk-linux
+        ANDROID_BUILD_TOOLS: 25.0.2
+        APPLICATION_ID: net.squanchy.example
+        FABRIC_API_KEY: 0000000000000000000000000000000000000000 
```

Comment:
No, this has to match (like all other env variables) the ones the build looks for, otherwise the secrets plugin won't find them